src/workerd/api/node/util.c++

```diff
@@ -289,4 +289,16 @@ void UtilModule::processExitImpl(jsg::Lock& js, int code) {
   js.logWarning(kj::str(err.get(js, "stack"_kj)));
 }
 
+kj::String UtilModule::getProcessPlatform() const {
```

Comment:
Let's have this return `kj::StringPtr` and have these values be literal constants to avoid the extraneous allocation on every call.